app/test_split_integer.py

```diff
@@ -1,21 +1,57 @@
+import pytest
 from app.split_integer import split_integer
 
 
-def test_sum_of_the_parts_should_be_equal_to_value() -> None:
-    pass
+@pytest.mark.parametrize(
+    "value,number_of_parts,result",
+    [
+        (8, 1, 8),
+        (6, 2, 6),
+        (17, 4, 17),
+        (32, 6, 32)
```

Comment:
The expected result should be a list of integers representing the parts, not a single integer. For example, it should be `[5, 5, 5, 5, 6, 6]` instead of `32`.